lib/reverse_sentence.rb

```diff
@@ -1,4 +1,51 @@
 # A method to reverse the words in a sentence, in place.
-def reverse_sentence(my_sentence)
-  raise NotImplementedError
+def reverse_letters(my_string)
+
+  return my_string if my_string.length == 1
+
+  while my_string != nil
+    i = 0
+    j = (my_string.length - 1)
+    while i < j
+      temp = my_string[j]
+      my_string[j] = my_string[i]
+      my_string[i] = temp
+      j -= 1
+      i += 1
+    end
+    return my_string
+  end
+  return nil
+end
+
+def reverse_sentence(my_string)
```

Comment:
This looks the same as your reverse_words's algorithm. Are the tests passing for you?